src/modules/personenkontext/api/dbiam-personenkontext-workflow.controller.ts

```diff
@@ -216,6 +216,7 @@ export class DbiamPersonenkontextWorkflowController {
                 params.familienname,
                 params.organisationId,
                 params.rolleId,
+                params.personalnummer || undefined,
             );
```

Comment:
The usage of `||` will result in `undefined`, if the personalnummer is an empty string. I assume that was your intention, but I just wanted to clarify because type-coercions using `||` can be a mistake

Reply:
It is intended yes.